test/test_fx.py

```diff
@@ -42,6 +42,7 @@
 from fx.test_dce_pass import TestDCE  # noqa: F401
 from fx.test_fx_const_fold import TestConstFold  # noqa: F401
 from fx.test_fx_param_shape_control_flow import TestConstParamShapeInControlFlow  # noqa: F401
+from fx.test_pass_infra import TestPassManager  # noqa: F401
```

Comment:
do you need this line?

Reply:
I think I do in order to run `python test/test_fx.py TestPassManager`

Reply:
ic.